assets/php/examples/events_actions/javascript_alerts.tpl.php

```diff
@@ -17,7 +17,12 @@
 	javascript function on the page itself, so that the button has some javascript to perform.</p>
 	
 	<p>If you are interested in more advanced and flexible types of confirmation or prompts, take a look at the 
-	<a href="../../../../plugins/DialogBoxes/example/more_dialog_boxes.php">plugin that offers pre-built QDialogBox subclasses</a>.</p>
+		<?php if (file_exists('../../../../plugins/DialogBoxes/example/more_dialog_boxes.php')) { ?>
+		<a href="../../../../plugins/DialogBoxes/example/more_dialog_boxes.php">plugin that offers pre-built QDialogBox subclasses</a>.
```

Comment:
This is not a 3.0 plugin, and some dialog functionality has been added to 3.0. I will commit this, but we need to look at this again before releasing.